src/Avalonia.Native/AvaloniaNativePlatform.cs

```diff
@@ -26,9 +26,13 @@ class AvaloniaNativePlatform : IPlatformSettings, IWindowingPlatform
 
         public TimeSpan DoubleClickTime => TimeSpan.FromMilliseconds(500); //TODO
 
+        public Size TouchDoubleClickSize => new Size(16, 16);
```

Comment:
If the interface was commented you could add `<inheritdoc cref="ITouchPlatformSettings.TouchDoubleClickSize"/>`.

Reply:
You don't need to add inheritdoc with latest VS. It detects interface comments automatically

Reply:
True, might help to be clear though. The properties are not defined through the interface either.

Reply:
Added.